packages/vuetify/src/composables/date/DateAdapter.ts

```diff
@@ -37,6 +37,7 @@ export interface DateAdapter<T = unknown> {
   getDiff (date: T, comparing: T | string, unit?: string): number
   getWeekArray (date: T, firstDayOfWeek?: number | string): T[][]
   getWeekdays (firstDayOfWeek?: number | string): string[]
+  getWeek (date: T, firstDayOfWeek?: number | string, firstWeekMinSize?: number): number
```

Comment:
I know that I added new arguments to some of the typings but it was probably the wrong move. We need to figure out a way to get this information without needing to change the interface.